Release Cognitive Services Data Plane SDKs Round I (#4955)
Azure/azure-sdk-for-js: sdk/cognitiveservices/cognitiveservices-textanalytics/README.md

#### nodejs - Authentication, client creation and detectLanguage  as an example written in TypeScript.

##### Install @azure/ms-rest-azure-js

```bash
npm install @azure/ms-rest-azure-js
```

##### Sample code

> The following sample lets you detect the langauges of input data. In additions, it provides data such as Characters count, transaction count, etc. To know more, Refer the [Azure Documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview)

Reviewer: ```suggestion
The following sample detects the langauge in the provided text. In addition, it provides data such as Characters count, transaction count, etc. To know more, refer to the [Azure Documentation on Text Analytics](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview)
```
